docs/framework/wpf/graphics-multimedia/how-to-control-the-fill-of-a-composite-shape.md

```diff
@@ -19,39 +19,39 @@ The <xref:System.Windows.Media.GeometryGroup.FillRule%2A> property of a <xref:Sy
   
  The following illustration shows the shape created in the previous example.  
   
- ![Screenshot: FillRule Property of EvenOdd](./media/fillruleevenoddfirstone.png "FillRuleEvenOddFirstOne")  
+ ![Diagram showing the FillRule property of EvenOdd.](./media/how-to-control-the-fill-of-a-composite-shape/fillrule-evenodd-property.png)  
   
  In the illustration above, notice that the center and 3rd ring are not filled. This is because a ray drawn from any point within either of those two rings passes through an even number of segments. See illustration below:  
   
- ![Diagram: FillRule property value of EvenOdd](./media/fillruleevenodd2.png "FillRuleEvenOdd2")  
+ ![Diagram showing the EvenOdd rays drawn in the circle.](./media/how-to-control-the-fill-of-a-composite-shape/fillrule-evenodd-rays.png)  
   
  **NonZero:** This rule determines whether a point is in the fill region of the path by drawing a ray from that point to infinity in any direction and then examining the places where a segment of the shape crosses the ray. Starting with a count of zero, add one each time a Segment crosses the ray from left to right and subtract one each time a path segment crosses the ray from right to left. After counting the crossings, if the result is zero then the point is outside the path. Otherwise, it is inside.  
   
  [!code-xaml[GeometriesMiscSnippets_snip#FillRuleNonZeroValueEllipseGeometry](~/samples/snippets/xaml/VS_Snippets_Wpf/GeometriesMiscSnippets_snip/XAML/FillRuleExample.xaml#fillrulenonzerovalueellipsegeometry)]  
   
  Using the example above, a value of <xref:System.Windows.Media.FillRule.Nonzero> for <xref:System.Windows.Media.GeometryGroup.FillRule%2A> gives the following illustration as a result:  
   
- ![Screenshot: FillRule value of NonZero](./media/fillrulenonzero1.png "FillRuleNonZero1")  
+ ![Illustration showing the FillRule value of Nonzero.](./media/how-to-control-the-fill-of-a-composite-shape/fillrule-value-nonzero.png)  
   
  As you can see, all the rings are filled. This is because all the segments are running in the same direction and so a ray drawn from any point will cross one or more segments and the sum of the crossings will not equal zero. For example, in the illustration below, the red arrows represent the direction the segments are drawn and the white arrow represents an arbitrary ray running from a point in the innermost ring. Starting with a value of zero, for each segment that the ray crosses, a value of one is added because the segment crosses the ray from left to right.  
```

Comment:
```suggestion
 As you can see, all the rings are filled. This is because all the segments are running in the same direction and so a ray drawn from any point will cross one or more segments and the sum of the crossings will not equal zero. For example, in the following illustration, the red arrows represent the direction the segments are drawn and the white arrow represents an arbitrary ray running from a point in the innermost ring. Starting with a value of zero, for each segment that the ray crosses, a value of one is added because the segment crosses the ray from left to right.  
```